codecov.yml

```diff
@@ -0,0 +1,12 @@
+codecov:
+  require_ci_to_pass: false
+
+fixes:
+  - "/home/circleci/project/::"        # reduce root e.g., "before/path/" => "path/"
+
+coverage:
+  status:
+    project:
+      default:
+        target: 95%
```

Comment:
Current coverage is 94.3%, wouldn't this fail

Reply:
I'm changing it to "auto" and threshold 0% which means not less then main, but we need to merge and have it run on main for that...